### ISSUE TITLE
Antisteal

### ISSUE THREAD

**Issue by KosmX** [KosmX/emotes#134]
https://github.com/i-am-cal/antisteal

I may implement it

**Comment by CamperSamu** [comment]
I don't think that implementing DRM in mods is a good idea, but it's not really a bad idea too...
I don't know about this, it can be limiting at times...

**Comment by dima-dencep** [comment]
Since modrinth is blocked in Russia, it may become very important to maintain income from the mod
